### <center> Exp 5 FLOW MEASUREMENT (VENTURIMETER, ORIFICEMETER & ROTAMETER)

### **Objective**:
* To measure discharge through Venturimeter, Orificemeter and Rotameter.

### **Aim**:
* To determine the co-efficient of discharge through Venturimeter &amp; Orificemeter.
* To calibrate the Rotameter.

### **Introduction**:
If a constriction is placed in a closed channel carrying a stream of fluid, there will be
increase in velocity, and hence increase in Kinetic Energy, at the constriction, from an energy
balance, as given by Bernoulli’s Theorem, there must be a corresponding reduction in pressure.
Rate of discharge from the constriction can be calculated by knowing this pressure reduction, the
area available for flow at the constriction, the density of fluid, and the Co-efficient of discharge.
The last named is defined as the ratio of actual flow to the theoretical flow and makes allowance
for stream contraction and frictional effects.

### **Theory**:

**VENTURIMETER**: It consists of:
1. An inlet section followed by a convergent cone.
2. A Cylindrical Throat.
3. A gradually divergent cone.
The inlet section of the Venturimeter is of the same diameter as that of the pipe, which is
followed by a convergent cone. The convergent cone is a short pipe, which tapers from the
original size of the pipe to that of the Throat of the Venturimeter. The Throat of the Venturimeter
is a short parallel side tube having its cross-sectional area smaller than that of the pipe. The
divergent cone of the Venturimeter is gradually diverging pipe with its cross-sectional area
increasing from that of the Throat to the original size of the pipe. At inlet section &amp; Throat of the
Venturimeter, pressure taps are provided.

**ORIFICEMETER**:
An Orificemeter consists of a flat circular plate with a circular hole called Orifice, which
is concentric with the pipe axis.

**ROTAMETER**:
The Rotameter is a variable – area meter that consists of an enlarging transparent tube
and a metering “float” (actually heavier than the liquid) that is displaced upward by the upward
flow fluid through the tube. The tube is graduated to read the flow directly. Notches in the float
cause it to rotate and thus maintain a central position in the tube.

### **Procedure**:
**Starting Procedure**:
1. Close the drain valves in the system.
2. Fill sump tank ¾ with clean water and ensure that no foreign particles are there.
3. Close all control valves given on the water line and open by-pass valve.
4. Close all Pressure Taps of Manometer connected to Venturimeter &amp; Orificemeter.
5. Ensure that On/Off Switch given on the Panel is at OFF position.
6. Now switch on the main power supply.
7. Switch on the pump.
8. Operate the control valve to regulate the flow of water in the desired test section.
9. Open the pressure taps of manometer of related test section, very slowly to avoid the blow of water on manometer fluid.
10. Now open the air release valve provided on the manometer and slowly release the air from manometer.
11. When there is no air bubbles in the manometer, close the air release valves.
12. Adjust water flow rate in desired section with the help of control valve and record the manometer reading.
13. Measure the flow of water, discharged through desired test section, using stopwatch (30 s) and measuring tank.
14. Repeat the above procedure for different flow rates of water, operating controlvalve and by-pass valve.

**Calibration of Rotameter**:
1. Close the ball valves provided in the Venturimeter and Orificemeter pipelines.
2. Open the ball valve provided in the Rotameter pipeline.
3. Now switch on the main power supply and switch on the pump.
4. Set the flow rate with the help of by pass and flow control valves provided in Rotameterpipeline.
5. Measure the discharge with the help of measuring tank and stopwatch.
6. The actual discharge, verify the set value of rotameter.
7. Repeat the same procedure for different flow rates.

**Closing Procedure**:
1. When the experiment is over, close all manometers pressure taps first and then switch off the pump.
2. Switch off power supply to panel.
3. Drain all the tanks.

### **Formula**

For both Venturimeter &; Orifice meter:

* $Head \; Losses$,
    
   $$H=\frac{h_{1}-h_{2}}{100}\left(\frac{\rho_{m}}{\rho_{W}}-1\right),\; m\;of\;water $$ 

* $Theoritical \; discharge$ 
        
   $$Q_{t}=\frac{a_{1}a_{2} \sqrt{2 g H}}{\sqrt{a_{1}^{2}-a_{2}^{2}}}\; , \;\; in\; m^{2}s^{2}$$


* $Actual \; discharge$ 
    
$$
\begin{aligned}
&Q_{a}=\frac{A^{*} R}{t} \mathrm{~m}^{3} / \mathrm{s} \\
&\boldsymbol{R}=\frac{R_{1}-R_{2}}{100}
\end{aligned}
$$

* $Coefficient\; of \;discharge$
    
    $$C_{d}=\frac{Q_{a}}{Q_{t}}$$


For Rotameter:

 * $Actual \;discharge$
    
    $$
\begin{aligned}
&Q_{A}=\frac{A^{*} R}{t}^{*} 3600^{*} 1000\;\; LPH \\
&R=\frac{R_{1}-R_{2}}{100} m
\end{aligned}
$$

### **Observations & Calculations**:

#### FOR VENTURIMETER
| S.No. |        h_1       cm |        h_2          cm  |       R_1        |       R_2      |   t  |
|:-----:|:-------------------:|:-----------------------:|:----------------:|:--------------:|:----:|      
|   1   |         10.4        |            14           |        20        |      32.6      |  30  |
|   2   |         10.5        |           13.8          |        20        |      32.3      |  30  |
|   3   |         11.2        |            13           |        20        |      28.2      |  30  |
                     

#### FOR ORIFICEMETER
| S.No. |        h_1       cm |        h_2          cm  |       R_1        |       R_2      |   t  |
|:-----:|:-------------------:|:-----------------------:|:----------------:|:--------------:|:----:|      
|   1   |         8.7         |         15.8            |        20        |      31.8      |  30  |
|   2   |         9.6         |         14.7            |        20        |      30.5      |  30  |
|   3   |         10.2        |         14.1            |        20        |      29.1      |  30  |

#### FOR ROTAMETER
| S.No. | Q_R  (LPH) |       R_1        |       R_2      |      t        |
|:-----:|:----------:|:----------------:|:--------------:|:-------------:|
|   1   |     225    |        20        |      21.8      |       30      |
|   2   |     200    |        20        |      21.7      |       30      |
|   3   |     145    |        20        |      21.2      |       30      |
|   4   |     135    |        20        |      21.1      |       30      |
|   5   |     95     |        20        |      20.8      |       30      |

In [1]:
import numpy as np

In [2]:
#common data
rho_m = 13600       #manometer fluid
rho_w = 1000        #water density 
g = 9.81            #acceleration due to gravity
A = 0.1             #Area of tank

In [20]:
#VENTURIMETER
h1 = np.array([10.4, 10.5, 11.2])
h2 = np.array([14,13.8,13])
R1 = np.array([20, 20, 20])
R2 = np.array([32.6, 32.3, 28.2])
t = np.array([30, 30, 30])
#DATA FOR VENTURIMETER
d1 = 0.028
d2 = 0.014
a1 = 6.157e-4
a2 = 1.539e-4 
#calculations
R = np.abs(R1 - R2)/100
H_vent = (np.abs(h1 - h2))*((rho_m/rho_w) -1)/100               #Head loss
Qt_vent = (a1*a2*np.sqrt(2*g*H_vent))/(np.sqrt(a1**2 - a2**2))  #Theoritical discharge
Qa_vent = A*R/t                                                 #Actual discharge
Cd_vent = Qa_vent/Qt_vent                                       #Co-efficient of discharge
print(Cd_vent,Qa_vent,Qt_vent)

[0.8857578  0.90311662 0.81521712] [0.00042    0.00041    0.00027333] [0.00047417 0.00045398 0.00033529]


In [21]:
#Orificemeter
h1 = np.array([8.7, 9.6, 10.2])
h2 = np.array([15.8,14.7,14.1])
R1 = np.array([20, 20, 20])
R2 = np.array([31.8, 30.5, 29.1])
t = np.array([30, 30, 30])
#DATA FOR ORIFICEMETER
d1 = 0.028
d2 = 0.014
a1 = 6.157e-4 
a2 = 1.539e-4
#calculations
R = np.abs(R1-R2)/100
H_ori = np.abs((h1 - h2))*((rho_m/rho_w) -1)/100              #Head loss
Qt_ori = (a1*a2*np.sqrt(2*g*H_ori))/(np.sqrt(a1**2 - a2**2))  #Theoritical discharge
Qa_ori = A*R/t                                                #Actual discharge
Cd_ori = Qa_ori/Qt_ori                                        #Co-efficient of discharge
print(Cd_ori,Qa_ori,Qt_ori)

[0.59067491 0.6201545  0.6146173 ] [0.00039333 0.00035    0.00030333] [0.0006659  0.00056438 0.00049353]


In [37]:
#ROTAMETER
Q_R = np.array([225, 200, 145, 135, 95])
R1 = np.array([20, 20, 20, 20, 20])
R2 = np.array([21.8, 21.7, 21.2, 21.1, 20.8])
t = np.array([30, 30, 30, 30, 30])
R = np.abs(R2-R1)/100
Qa_rota = A*R*3600*1000                                       #Actual discharge
print(Qa_rota)

[6480. 6120. 4320. 3960. 2880.]


### Calculation Table:

**For venturimeter:**

| S.No. |  Q_a  (m3/s)   |      Q_t  (m3/s)       |      C_d      |
|:-----:|:--------------:|:----------------------:|:-------------:|
|   1   |     0.00042    |         0.00047        |    0.88575    |
|   2   |     0.00041    |         0.00045        |    0.90311    |
|   3   |     0.00027    |         0.00033        |     0.8152    |

**For orificemeter:**

| S.No. |  Q_a  (m3/s)   |      Q_t  (m3/s)       |      C_d      |
|:-----:|:--------------:|:----------------------:|:-------------:|
|   1   |    0.000393    |         0.00066        |    0.59067    |
|   2   |     0.00035    |        0.000564        |    0.62015    |
|   3   |     0.00030    |         0.00049        |    0.61461    |

**For rotameter:**

| S.No. |  Q_R  (LPH)   |      Q_A  (LPH)        |
|:-----:|:-------------:|:----------------------:|
|   1   |      225      |          6480          |
|   2   |      200      |          6120          |
|   3   |      145      |          4320          |
|   4   |      135      |          3960          |
|   5   |       95      |           20           |

### **Conclusions/Inferences**:


For the venturi meter, the actual value obtained was closer to the theoritical calculations. This is due to lower pressure drop that results from its streamlined shape which almost eliminates boundary-layer separation and thus form drag is negligible. It has a converging and a diverging part, pressure lost in the converging part is attained back in the diverging part of the meter. This meter is good for measuring high pressure flows and appreciable energy recovery.
The head loss is maximum for the orifice meter. To be concluded, venturi meter is more accurate compared to orifice meter and rotameter.

### **Industrial Applications**:

The oil and gas industries and refinery are largely dependent on differential pressure flow meters for their liquid flow measurement applications. The differential pressure flow meters operate by restricting the cross-sectional area of a flowing fluid. A pressure drop is caused across the constriction when the flow area is restricted and this drop in pressure is caused by a change in the velocity of the fluid. A differential pressure flow meter is the best choice at times when no moving parts are required or when prompt response time is needed. These fluid flow meters are found in industrial applications to calculate the fuel output, plain water measurement testing or in specialty chemical manufacturing. They are also used in laboratories for calculation and controlling the flow of gases while separating or mixing them with the help of chromatography. After analyzing the pipe's size, nature of liquids and flow conditions, the users must select the right type. Several types of differential pressure meters are available. These include -

1. Flow Tubes
2. Orifices
3. Venturi Tubes
4. Target Meters
5. Flow Nozzles
6. Elbow-Tap Meters
7. Variable-Area Meters
8. Rotameters